doc/modules/mod_mam.md

```diff
@@ -158,3 +157,9 @@ You can change the default settings using extra parameters:
               ]}
        ]}.
 ```
+
+### Backend metrics
+
+* `[global, backends, mod_mam, lookup]` - A time of archive lookup operation.
```

Comment:
time it takes to archive the lookup operation

or, if you mean a point in time rather than duration:

time of the archive lookup operation.